public/index.html

```diff
@@ -6,6 +6,6 @@
 </head>
 <body>
   <div id="app"></div>
-  <script src="/bundle.js"></script>
+  <script src="./bundle.js"></script>
```

Comment:
This caused a lot of headache when logout function wasn't working. Please don't edit this!